
From: Andi Kleen <ak@suse.de>

From: Tom Duffy

Fix compilation with O= on x86-64

Apparently works around some hard to fix kbuild issue.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/x86_64/pci/Makefile |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/x86_64/pci/Makefile~fix-o=-compilation-on-x86-64 arch/x86_64/pci/Makefile
--- 25/arch/x86_64/pci/Makefile~fix-o=-compilation-on-x86-64	2004-08-08 15:56:19.559879424 -0700
+++ 25-akpm/arch/x86_64/pci/Makefile	2004-08-08 15:56:19.563878816 -0700
@@ -3,7 +3,7 @@
 #
 # Reuse the i386 PCI subsystem
 #
-CFLAGS += -I arch/i386/pci
+CFLAGS += -Iarch/i386/pci
 
 obj-y		:= i386.o
 obj-$(CONFIG_PCI_DIRECT)+= direct.o
_
